tools/udp-test-server/Dockerfile

```diff
@@ -18,7 +18,7 @@ RUN --mount=type=cache,target=/workspace/target/ \
 RUN --mount=type=cache,target=/workspace/target/ \
     cp /workspace/target/x86_64-unknown-linux-musl/release/udp-test-server /workspace/udp-test-server
 
-FROM gcr.io/distroless/static
+FROM alpine
```

Comment:
Why are we moving this one from distroless to alpine?

Reply:
The commit for this included an explanation:

d783a56dc72de73b5b04b7099f3a09df175855be

```
This enables the ability to add tools, such as tcpdump
and do some debugging.
```